css/revops.css

```diff
@@ -69,6 +69,44 @@ p {
   display:  flex;
   justify-content: space-evenly;
 }
+.revops-section__container--grid {
+  max-width: 1200px;
+  padding: 0 2rem 4rem;
+  text-align: left;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+}
+.revops-section__problem,
```

Comment:
As far as I can see in this case we could do with one element class and a modifier instead of `__solution`.

Or maybe we should have a Block specific to this layout, fx. `.revops-solution`, that contains `.__problem` and `.__solution` classes. Just a thought to simplify the layout and grid.

Reply:
Ah, my mistake, fixed 😊